customTests/azure_ib_write_bw_gdr.nhc

```diff
@@ -13,15 +11,16 @@ function run_ib_bw_gdr(){
    SKU=$2
    PORTARG=$3
    device=$4
+ 
    #NUMA mapping
-
    GPU_NUMA=( 0 0 0 0 0 0 0 0 )
    if [[ "$SKU" == "nd96asr_v4" || "$SKU" == "nd96amsr_v4" ]]; then
       GPU_NUMA=( 1 1 0 0 3 3 2 2 )
    elif [[ "$SKU" == "nd96isr_v5" ]]; then
       GPU_NUMA=( 0 0 0 0 1 1 1 1 )
    fi
    MESSAGE_SIZE=8388608
+
    IB_WRITE_BW=`basename $IB_WRITE_BW_EXE_PATH`
    IB_WRITE_BW_ARGS="-p $PORTARG -s $MESSAGE_SIZE -x 0 -F --report_gbits  --perform_warm_up"
```

Comment:
Why aren't we using bidirectional (-b)?

Reply:
Well we are testing the HCA against itself... would bidirectional add any value?

Reply:
No, good point!